File: Sample/NendSDK_Sample/FullBoardAdContentViewController.m
//
//  FullBoardAdContentViewController.m
//  NendSDK_Sample
//
//  Copyright © 2017年 F@N Communications. All rights reserved.
//

#import "FullBoardAdContentViewController.h"

@interface FullBoardAdContentViewController ()

@property (weak, nonatomic) IBOutlet UILabel *pageLabel;

@end

@implementation FullBoardAdContentViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
    
    _pageLabel.text = self.number;
}

@end

File: Sample/NendSDK_Swift_Sample/FullBoardAdContentViewController.swift
//
//  FullBoardAdContentViewController.swift
//  NendSDK_Sample
//
//  Copyright © 2017年 F@N Communications. All rights reserved.
//

import UIKit

class FullBoardAdContentViewController: UIViewController {
    
    @IBOutlet fileprivate weak var pageLabel: UILabel!
    var number: String?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        self.pageLabel.text = number;
    }
}
